projects/igniteui-angular/src/lib/grids/grid/grid-filtering-advanced.spec.ts

```diff
@@ -1389,6 +1389,22 @@ describe('IgxGrid - Advanced Filtering #grid - ', () => {
             // Verify no filtered data
             expect(grid.filteredData).toBe(null);
         }));
+
+    });
+
+    describe('Expression tree serialization - ', () => {
+        it('should correctly filter with a deserialized expression tree.', fakeAsync(() => {
+            const errorSpy = spyOn(console, 'error');
+            let fix = TestBed.createComponent(IgxGridAdvancedFilteringSerializedTreeComponent);
```

Comment:
I dont see anything by searching for `IgxGridAdvancedFilteringSerializedTreeComponent`, and tests also fail because of that.